README.md

```diff
@@ -68,6 +69,23 @@ $ sudo find / -name qmake
 ## Run:
 ``` ./start.sh [options] [script file name] ```
 
+## Documentation generation
+### Download and install [ldoc](stevedonovan.github.io/ldoc/manual/doc.md.html)
+```
+sudo apt install luarocks
+sudo luarocks install luasec
+sudo luarocks install penlight
+sudo luarocks install ldoc
```

Comment:
@pestOO luasec penlight discount are dependencies for ldoc?
So if yes, why discount is installed after ldoc?